internal/ignition/ignition.go

```diff
@@ -682,7 +683,8 @@ func (g *installerGenerator) updateBootstrap(ctx context.Context, bootstrapPath
 			g.fixMOTDFile(&config.Storage.Files[i])
 		case isBMHFile(&config.Storage.Files[i]):
 			// extract bmh
-			bmh, err := fileToBMH(&config.Storage.Files[i]) //nolint,shadow
+			var bmh *bmh_v1alpha1.BareMetalHost
```

Comment:
not sure why to change this, it makes review a little bit harder. Lets not change stuff that is not required by PR or at least  lets change it in another commit.

Reply:
actually, changes in this pr brought these lint changes.  Next time will do it in separate commits. Thanks